<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAccountClass</class>
 <widget class="QDialog" name="CreateAccountClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create New Account</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="TitleBarWidget" name="m_TitleBarWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>No Registration Required.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Just Enter A Online User Name and Log In!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_UserNameLabel">
     <property name="text">
      <string>User Name: (4 to 27 characters)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_UserNameEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maxLength">
      <number>27</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_MoodMessageLabel">
     <property name="text">
      <string>Mood Message: (Up to 31 characters)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_MoodMessageEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Let's Communicate!</string>
     </property>
     <property name="maxLength">
      <number>31</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Age:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_AgeComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Preferred Content:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_ContentComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maxCount">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_GenderComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Primary Language:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_LanguageComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="m_LoginButton">
     <property name="text">
      <string>Login!</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>160</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="BottomBarWidget" name="m_BottomBarWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header>TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BottomBarWidget</class>
   <extends>QWidget</extends>
   <header location="global">BottomBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
